### PR TITLE
Fix possible null reference of RenderHandler

### PR DESCRIPTION
**Pull request by DamirAinullin**
Fix possible null reference of RenderHandler in second part of condition.

### CODE REVIEW

**Comment by amaitland** [comment]
Please include a description and simplify the title. 

Looking at the code again and could probably be rewritten to use **is** checks. Happy to go with this for now :+1:

**Comment by DamirAinullin** [comment]
@amaitland Done.

File: CefSharp.Wpf/ChromiumWebBrowser.cs [hunk 1]
```diff
@@ -1576,18 +1576,21 @@ private void PresentationSourceChangedHandler(object sender, SourceChangedEventA
                     }
 
                     //Ignore this for custom bitmap factories
-                    if (RenderHandler != null && RenderHandler.GetType() == typeof(WritableBitmapRenderHandler) || RenderHandler.GetType() == typeof(InteropBitmapRenderHandler))
+                    if (RenderHandler != null)
```
Comment (amaitland) [inline CefSharp.Wpf/ChromiumWebBrowser.cs]: This is now redundant and can be removed

Comment (DamirAinullin) [reply CefSharp.Wpf/ChromiumWebBrowser.cs]: removed

**Pull request merged**